egs/wsj/s5/local/nnet3/run_splstm.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+# this is a state preserving lstm script
+# LSTM script runs for more epochs than the TDNN script
+# and each epoch takes twice the time
+
+# At this script level we don't support not running on GPU, as it would be painfully slow.
+# If you want to run without GPU you'd have to call lstm/train.sh with --gpu false
+
+stage=0
+train_stage=-10
+affix=state_preserving_shift
+common_egs_dir=
+
+# LSTM options
+lstm_delay=" -1 -2 -3 "
+label_delay=5
+num_lstm_layers=3
+cell_dim=1024
+hidden_dim=1024
+recurrent_projection_dim=256
+non_recurrent_projection_dim=256
```

Comment:
in the options highlight what is different from a normal LSTM. In this case chunk_width and minibatch_chunk_size are different so place them separately and describe how they are going to be interpreted in the new script.